canonical-web-and-design/landscape-brochure.internal
# Ready site for release

## Done
- Updated the content on the "On prem" page to match the live site
- Removed login from the global navigation
- Added a login navigation item to the right of the navigation
- Bumped the versions of global-nav and cookie-policy to the latest versions

## QA
- Load the demo
- Check that the global nav has no login menu
- Check there is a login item in the navigation that links to the live site
- Check the cookie-policy and global-nav work as expected

Fixes https://github.com/canonical-web-and-design/landscape-brochure.internal/issues/23
Fixes https://github.com/canonical-web-and-design/landscape-brochure.internal/issues/24
Fixes https://github.com/canonical-web-and-design/landscape-brochure.internal/issues/12
